spec/ddtrace/contrib/rails/analytics_spec.rb

```diff
@@ -12,7 +12,7 @@
       # This is because Rails instrumentation normally defers patching until #after_initialize
       # when it activates and configures each of the Rails components with application details.
       # We aren't initializing a full Rails application here, so the patch doesn't auto-apply.
-      c.use :action_pack
+      c.use :action_pack, configuration_options
```

Comment:
This test started to fail because it was not configuring a test tracer, using the global tracer instead.